tests/unit/amo/components/TestLandingAddonsCard.js

```diff
@@ -106,6 +107,77 @@ describe(__filename, () => {
     expect(root.find(AddonsCard)).toHaveProp('footerLink', null);
   });
 
+  it('hides the footer link when there are less add-ons than LANDING_PAGE_THEME_COUNT', () => {
+    const addons = [
+      createInternalAddon({
```

Comment:
This test is assuming that `1` (the number of add-ons you're creating) is less than `LANDING_PAGE_THEME_COUNT`, but the test doesn't know that to be the case. I think you need to use `LANDING_PAGE_THEME_COUNT`-1 in order to figure out how many add-ons to create.